themes/tiddlywiki/vanilla/base.tid

```diff
@@ -417,6 +421,10 @@ a.tc-tiddlylink-external:hover {
 ** Buttons
 */
 
+button {
```

Comment:
Even though this is about buttons, it should go up at the top where we have gathered the other style declarations that target all instances of HTML elements.

Reply:
ok, I'll move it